python/bashrc

```diff
@@ -0,0 +1,19 @@
+VIRTUAL_ENV_PATH=$(pwd)/venv
+
+if (tty -s); then
```

Comment:
Why copying instead of executing `virtualenv venv` and `source venv/bin/activate`?
Another option is to use `pipenv`:
https://github.com/realpython/python-guide/blob/master/docs/dev/virtualenvs.rst#installing-packages-for-your-project
Also, I think that managing the virtual environment in the `bashrc` file is too magic. The sample should be clear on how to manage the dev environment. I would move the steps to the guide and explain the advantages of caching the virtual environment (no need to build docker images in your inner loop).

Reply:
Creating the virtual environment takes about 3-5 seconds, so it's a pretty bad experience. If we don't do this, then we should just tell the dev to go through to the hassle of creating the virtual env.

Reply:
It is done once. Think about the day to day workflow.